contracts/gateway/AxelarAmplifierGatewayProxy.sol

```diff
@@ -5,6 +5,8 @@ pragma solidity ^0.8.0;
 import { Proxy } from '../upgradable/Proxy.sol';
 
 contract AxelarAmplifierGatewayProxy is Proxy {
+    bytes32 private constant CONTRACT_ID = keccak256('axelar-amplifier-gateway-proxy');
```

Comment:
```suggestion
    bytes32 public constant CONTRACT_ID = keccak256('axelar-amplifier-gateway');
```
this is confusing since there's already a method called `contractId` with a different id. just make it public instead and use it internally to avoid duplication